src/OrchardCore.Build/OrchardCore.Commons.props

```diff
@@ -3,6 +3,7 @@
   <Import Project="Dependencies.props" />
 
   <PropertyGroup>
+    <LangVersion>latestMajor</LangVersion>
```

Comment:
I'd use a specific version, i.e. `10.0`, so it doesn't depend on the SDK you have installed, and we can update it deliberately.

Reply:
At the beginning I was planned to set it to `10.0` but I prefered to use `latestMajor` to make sure that we no longer need to update it

> so it doesn't depend on the SDK you have installed

Right, but while each OC version required specific framework it fine IMHO

Reply:
We've multi-targeted frameworks before and most possibly we'll continue to always target the latest version and the latest LTS one, so the framework alone is not a decisive factor.

Reply:
No problem to change it to `10.0`